src/tox_faster/main.py

```diff
@@ -1,18 +1,13 @@
-import sys
-from argparse import ArgumentParser
-from importlib.metadata import version
+from os import environ
 
+import pluggy
 
-def hello_world():
-    return "Hello, world!"
+hookimpl = pluggy.HookimplMarker("tox")
 
 
-def entry_point():  # pragma: nocover
-    parser = ArgumentParser()
-    parser.add_argument("-v", "--version", action="store_true")
+@hookimpl
+def tox_runenvreport(venv, action):  # pylint:disable=unused-argument
+    if "CI" in environ:
+        return None
 
-    args = parser.parse_args()
-
-    if args.version:
-        print(version("tox-faster"))
-        sys.exit()
+    return []
```

Comment:
Is the default template assuming you want to make a command? I'm assuming that's what we're removing here instead of this being a pure green commit.

It might make for nicer reviews to assume a bit less about what's being created? I assume 9/10 we'd be deleting this whenever we start a project?

Reply:
This is actually a bug in the cookiecutter. Whether or not you want a CLI is one of the `cookiecutter.json` question and defaults to no, but this `entry_point()` function is still generated regardless. Needs an `{% if %}` around it in the cookiecutter

Reply:
https://github.com/hypothesis/cookiecutters/issues/35